themes/default/layouts/page/cloud-engineering-summit.html

```diff
@@ -0,0 +1,110 @@
+{{ define "hero" }}
+    {{ partial "hero" (dict "title" .Params.title )}}
+{{ end }}
+
+{{ define "main" }}
+    <section class="relative my-16">
+        <div class="shape-background cloud-engineering-summit-big-dots">
+            <div class="relative w-full h-full">
+                <div class="circle-left"></div>
+                <div class="circle-right"></div>
+            </div>
+        </div>
+
+        <div class="container mx-auto">
+            <img class="mx-auto mb-6" src="/images/cloud-eng-summit-purple.png" />
+            <pulumi-date-countdown
+                date-string="2021-10-20T09:00:00-07:00"
+                text-class="text-3xl text-black text-center"
+                value-label-class="text-xl"
+                countdown-over-text="The Cloud Engineering Summit has started!"
+            >
+            </pulumi-date-countdown>
+            <div class="max-w-4xl mx-auto text-center">
+                <p class="mt-10">
+                    The Cloud Engineering Summit is a day of learning and speed talks for cloud practitioners
+                    and anyone who's passionate about cloud infrastructure, modern applications, and everything
+                    in between. Cloud engineering applies standard software engineering practices and tools
+                    uniformly across infrastructure, development, and security teams to tame the complexity of
+                    delivering and managing modern cloud applications.
+                </p>
+            </div>
+        </div>
+    </section>
+
+    <section class="container mx-auto text-center my-16">
+        <h3>Get notified when registration opens</h3>
+        <div class="flex items-center justify-center">
+            <pulumi-hubspot-form form-id="929d85ee-10e1-42c6-90d7-eab27730d9e5" class="newsletter newsletter-wide newsletter-dark-border"></pulumi-hubspot-form>
+        </div>
+    </section>
+
+    <section class="my-16 relative">
+        <div class="shape-background cloud-engineering-summit-small-dots">
+            <div class="relative w-full h-full">
+                <div class="dot-background-container">
+                    <div class="dot-background-overlay"></div>
+                    <div class="dot-background no-animate"></div>
+                </div>
+            </div>
+        </div>
+
+        <div class="container mx-auto">
+            <h2 class="text-center">Summit Tracks</h2>
+            <div class="text-center max-w-4xl mx-auto">
+                <p class="mb-12">
+                    We're inviting community members, including infrastructure (DevOps, SREs, infra engineers),
```

Comment:
Without the parenthetical, this reads as `We're inviting community members, including infrastructure, platform engineers...`, so suggest putting `engineers` outside the parens:

```suggestion
                    We're inviting community members, including infrastructure (DevOps, SRE, infra) engineers,
```

Reply:
👍